packages-exp/app-check-exp/src/storage.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { uuidv4 } from './util';
+import { FirebaseApp } from '@firebase/app-exp';
+import { isIndexedDBAvailable } from '@firebase/util';
+import {
+  readDebugTokenFromIndexedDB,
+  readTokenFromIndexedDB,
+  writeDebugTokenToIndexedDB,
+  writeTokenToIndexedDB
+} from './indexeddb';
+import { logger } from './logger';
+import { AppCheckTokenInternal } from './types';
+
+/**
+ * Always resolves. In case of an error reading from indexeddb, resolve with undefined
```

Comment:
Are these public-facing comments in this file?

Reply:
No, the only public strings here are the logger.warn and console.log messages.